tools/tokenizer_playground.py

```diff
@@ -0,0 +1,29 @@
+from zeeguu.core.model.language import Language
+from zeeguu.core.util.text import tokenize_text_flat_array
+from zeeguu.api.app import create_app
+
+from pprint import pprint
+import os
+
+
+def clear_terminal():
+    os.system("cls" if os.name == "nt" else "clear")
+
+
+app = create_app()
+app.app_context().push()
+
+while True:
+    clear_terminal()
+    print("Welcome to the Zeeguu Tokenizer")
+    language = Language.find(input("Enter language code (ex. 'en'): ").lower())
+    text = input("Enter text: ")
```

Comment:
thanks for the playground.

it's interesting to see that a double quote (") at the beginning of a sentence is not considered `left_punct`, but a chevron («) is. But maybe it's not that surprising: how is the poor parser to know that this is a left or a right token when both have the same shape. 

Though you'd expect that “...” would fare better, though it doesn't: 

```
Got the following text:  “...”
“
{'is_left_punct': False,
 'is_like_email': False,
 'is_like_num': False,
 'is_like_url': False,
 'is_punct': True,
 'is_right_punct': True,
 'is_sent_start': True,
 'paragraph_i': 0,
 'sent_i': 0,
 'text': '“',
 'token_i': 0}
```